src/main/java/com/capitalone/dashboard/collector/PullRequestCollector.java

```diff
@@ -60,14 +59,22 @@
      * @param status            open/merged
      *
      * @return
-     * @throws EncryptionException
+     * @throws Exception
```

Comment:
Technically you are suppressing the Exception by just logging the message. Can you remove this line?

Reply:
done